content/html/concepts/quotations/quotations.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'Quotations'
+Description: 'In HTML, a quotation is a portion of text that is attributed to a specific person or source.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Quotations'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+In HTML, a quotation is a portion of text that is attributed to a specific person or source. HTML provides several ways to mark up quotations on a web page, using specific [tags](https://www.codecademy.com/resources/docs/html/tags) for short and long quotations.
+
+## Short Quotations
+
+Short quotations such as a short sentence or fragment should be set off with the [`<q>` tag](https://www.codecademy.com/resources/docs/html/elements/q). Here's an example of how to use the `<q>` tag:
+
+```html
+<p>According to Albert Einstein, <q>Imagination is more important than knowledge.</q></p>
+```
+
+In this example, the text "Imagination is more important than knowledge" is enclosed in the `<q>` tag to indicate that it's a quotation. The quotation marks around "Imagination is more important than knowledge" are automatically added by the browser.
+
+## Long Quotations
+
+For longer quotations the `[<blockquote>](https://www.codecademy.com/resources/docs/html/elements/blockquote)` tag is used in lieu of `<q>`. Here's an example of how to use the `<blockquote>` tag:
+
+```html
+<blockquote>
+  <p>"The greatest glory in living lies not in never falling, but in rising every time we fall. The way to get started is to quit talking and begin doing. Your time is limited, don't waste it living someone else's life. Don't be trapped by dogma, which is living the result of other people's thinking. Don't let the noise of other's opinions drown your own inner voice. And most important, have the courage to follow your heart and intuition. They somehow already know what you truly want to become. Everything else is secondary."</p> 
```

Comment:
```suggestion
  <p>"The greatest glory in living lies not in never falling, but in rising every time we fall. The way 
  to get started is to quit talking and begin doing. Your time is limited, don't waste it living someone 
  else's life. Don't be trapped by dogma, which is living the result of other people's thinking. Don't 
  let the noise of other's opinions drown your own inner voice. And most important, have the courage
  to follow your heart and intuition. They somehow already know what you truly want to become. 
  Everything else is secondary."</p> 
```

Reply:
I think it's preferable to break the lines similar to what I've done above, so, that the quote is visible all at once, and users don't have to scroll to see what the text says.

Additionally, the footer with the author's name is missing.